extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -355,5 +355,17 @@ describes.realWin(
         expect(el.getAttribute('tabindex')).to.equal('0');
       });
     });
+
+    describe('[type=system]', () => {
+      it('should not throw if navigator.share fails', async () => {
+        Object.defineProperty(env.win, 'navigator', {
+          value: {share: env.sandbox.spy(() => Promise.reject())},
+        });
+        const element = await getShare('system');
+        const implementation = await element.getImpl();
+        expect(() => implementation.handleActivation_()).to.not.throw();
```

Comment:
This line expects `handleActivation_()` not to throw _any_ error. We should either modify to specify the specific error message (it shouldn't throw with `/cancel/`) or we should ensure the other possible errors thrown by this method are not the ones causing this test to fail.